DigitalLearningSolutions.Web/Scripts/searchSortFilterAndPaginate/searchSortFilterAndPaginate.ts

```diff
@@ -332,3 +343,7 @@ export class SearchSortFilterAndPaginate {
     }
   }
 }
+
+function defaultVoidFunction(): void {
+  return undefined;

```

Comment:
I'm not sure it's permitted by the linter, but I'd find it neater to have an empty function rather than an undefined return. But if you think otherwise it's not important.

Reply:
Unfortunately, that was what I had first tried to go for at first, but the linter disagreed. This was the cleanest solution that I could find that pleased it.